packages/protocol/test/governance/voting/election.ts

```diff
@@ -472,6 +516,192 @@ contract('Election', (accounts: string[]) => {
             )
           })
         })
+
+        describe('when the voter is over maxNumGroupsVotedFor but can vote for additional groups', () => {
+          let newGroup: string
+          beforeEach(async () => {
+            await election.setAllowedToVoteOverMaxNumberOfGroups(true)
+            await mockLockedGold.incrementNonvotingAccountBalance(voter, value)
+            for (let i = 0; i < maxNumGroupsVotedFor.toNumber(); i++) {
+              newGroup = accounts[i + 2]
+              await mockValidators.setMembers(newGroup, [accounts[9]])
```

Comment:
Do we not have a helper that sets up a group? If not, we probably have similar logic in the setup to this whole suite, so might be good to deduplicate.